TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ftc16072/Mechanisms/Motor.java

```diff
@@ -0,0 +1,33 @@
+package org.firstinspires.ftc.teamcode.ftc16072.Mechanisms;
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.teamcode.ftc16072.QQTest.QQtest;
+import org.firstinspires.ftc.teamcode.ftc16072.QQTest.TestMotor;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Motor extends Mechanism {
```

Comment:
Consider changing this to `SampleMechanism` instead of `Motor` so that it isn't confusing

Reply:
removed